### DISCUSSION TITLE
Homepage should link highlighted features

### DISCUSSION THREAD

**Discussion by christianmahnke** [cantaloupe-project/cantaloupe#547] [General]
On the [main site](https://cantaloupe-project.github.io/) several features are marked in blue, like:
* "One human-friendly configuration file"
* "Dynamic, selective image and text watermarking"
* "Selectable resample filters, sharpening levels, and compression levels"
* and many more

These should either be linkes to the documentation of the latest version or not be displayed in another colour, since this indicates links. Consider changing to bold or underline if links aren't wanted.

**Comment by nickdos** [comment]
Agree - I tried clicking on those twice, so even when it didn't work for me the first time, I did it again a few minutes later.
